.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
there should be a lot more files ignored, e.g. everything in .settings and target. Why was the old .gitignore removed?